ceph_deploy/hosts/remotes.py

```diff
@@ -179,10 +187,11 @@ def listdir(path):
     return os.listdir(path)
 
 
-def makedir(path, ignored=None):
+def makedir(path, ignored=None, uid=-1, gid=-1):
     ignored = ignored or []
     try:
         os.makedirs(path)
+        os.chown(path, uid, gid);
```

Comment:
I think I would put it in an `else:` clause after the except, so it is only run if there is not an exception. So this becomes:

```
try:
    os.makedirs(...)
except ....:
    ...
else:
    os.chown(...)
```

os.chown() is recursive, right? :)